src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -569,15 +585,15 @@ export function DashboardTopNav({
       createType: i18n.translate('dashboard.solutionToolbar.markdownQuickButtonLabel', {
```

Comment:
For the quickButtons in general, can we also add a tooltip to each? I know they have a `title` property, but the EuiTooltip component would make it stand out more.

Reply:
There isn't a way currently to add a tooltip around the individual buttons of a button group in EUI, so all we can do is assign a title that appears on hover.